### ISSUE TITLE
Cannot build in Windows 10 using build instructions

### ISSUE THREAD

**Issue by bjwest** [Chatterino/chatterino2#3406]
### Checklist

- [X] I'm reporting a problem with Chatterino
- [X] I've verified that I'm running a recent nightly build or the *newest* stable release
- [X] I've looked for my problem on the [wiki](https://wiki.chatterino.com/Help/)
- [X] I've searched the [issues](https://github.com/Chatterino/chatterino2/issues?q=) for similar looking reports

### Describe your issue

Following the Windows build instructions while installing both Visual Studio and Qt Creator for the fist time, build fails in both with the following errors:

Visual Studio - 

```
Severity	Code	Description	Project	File	Line	Suppression State
Error		CMake Error at C:\Users\Johnny\Source\Repos\chatterino2\CMakeLists.txt:43 (find_package):
  By not providing "FindQt5.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "Qt5", but
  CMake did not find one.

  Could not find a package configuration file provided by "Qt5" with any of
  the following names:

    Qt5Config.cmake
    qt5-config.cmake

  Add the installation prefix of "Qt5" to CMAKE_PREFIX_PATH or set "Qt5_DIR"
  to a directory containing one of the above files.  If "Qt5" provides a
  separate development package or SDK, be sure it has been installed.	chatterino	C:\Users\Johnny\Source\Repos\chatterino2\CMakeLists.txt	43	

```
  
Qt Creator - 

```
C:\local\boost\boost\asio\ssl\detail\openssl_types.hpp:23: error: C1083: Cannot open include file: 'openssl/conf.h': No such file or directory
```

Also, instructions are outdated.  OpenSSL 1.0 is no longer available for download, and the download links provided lead to an error page.  I installed OpenSSL 1.1.1m instead.

### OS and Chatterino Version

Windows 10 64, d21bdc6

**Comment by LosFarmosCTL** [comment]
What do you mean the OpenSSL links dont work? They work fine for me. The Qt creator build issue is definitely because of missing OpenSSL 1.0 files.

Also, building in visual studio will probably just not work in this state, since it uses CMake and not qmake which need to be configured properly to use Qt libraries in VS.

**Comment by Felanbird** [comment]
> What do you mean the OpenSSL links dont work? They work fine for me. 

The OpenSSL `1.1.1l` link is dead, the `1.0.2u` still works.

**Comment by LosFarmosCTL** [comment]
> The OpenSSL 1.1.1l link is dead, the 1.0.2u one* still works.

Huh yeah you are right they both worked for me just a few days ago when had to build on windows. The alternative link should still work though ig?

**Comment by zneix** [comment]
Building instructions don't mention Visual Studio - it seems to have issue with finding CMake anyway, use Qt Creator instead. 

> Also, instructions are outdated. OpenSSL 1.0 is no longer available for download

I suppose you've meant to type that OpenSSL 1.1 is dead, since you say that you had to install 1.1.1m later on, right?

> C:\local\boost\boost\asio\ssl\detail\openssl_types.hpp:23

I see that you have nested boost directory (`C:\local\boost\boost` should rather be `C:\local\boost` instead), maybe that's the issue?

**Comment by bjwest** [comment]
The link to 1.0.2u is now working, but it definitely sent me to a 404 page earlier today.  The alternate link does work, but it doesn't have a 1.0.2 version available.  After installing 1.0, I get a "'which' is not recognized as an internal or external command." in Qt Creator, same error in VS.  It does produce an executable, but it fails on multiple Qt5 .dlls when I try to run it.

I set up a fresh Windows 10 VM and followed the build instructions to the letter from downloading and installing Visual Studio 2022 through Producing standalone builds.  I still get the same "which" error but Chatterino runs fine.  I did realize in the previous setting, I deleted all the files in the Release directory and running windeployqt in the wrong order, deleting the Qt5 dlls from the Release directory in the process.  I went to the original VM, recompiled and performed the deleting and running windeployqt in the proper order and Chatterino ran fine.

TLDR: After installing OpenSSL 1.0 and correcting user error, everything works as should.

Note to zneix: Nested boost directory on first VM doesn't seem to cause problems, but I will fix it anyway.

**Comment by zneix** [comment]
> Note to zneix: Nested boost directory on first VM doesn't seem to cause problems, but I will fix it anyway.

:ok_hand: 

> I still get the same "which" error but Chatterino runs fine

Yup, that's expected. Just to clear things out, "`which` error" will be always present on windows since that's rather relevant only for Linux (and maybe MacOS?) environments where it looks up a command named `which` and tries to use `ccache` for speeding up builds.
